lib/engine_wrapper.py

```diff
@@ -754,6 +754,12 @@ def get_book_move(board: chess.Board, game: model.Game,
             try:
                 selection = polyglot_cfg.selection
                 min_weight = polyglot_cfg.min_weight
+                normalization = polyglot_cfg.normalization
+                entries = list(reader.find_all(board))
```

Comment:
To reduce redundancy, this line can be changed to `weights = [entry.weight for entry in reader.find_all(board)]`. Then the next line becomes shorter.